Makefile

```diff
@@ -406,6 +406,10 @@ test-bats-sync: check-linux binary binary-minimal bench check-skopeo $(BATS) $(N
 	$(BATS) --trace --print-output-on-failure test/blackbox/sync_docker.bats
 	$(BATS) --trace --print-output-on-failure test/blackbox/sync_replica_cluster.bats
 
+.PHONY: test-bats-delete-image
+test-bats-delete-image: check-linux binary binary-minimal bench check-skopeo $(BATS)
```

Comment:
Why do we need bench as prereq?
Also remove it from test-bats-sync

Reply:
I will refactor this.